t/unit-tests/generate-clar-decls.sh

```diff
@@ -14,6 +14,7 @@ do
 	suite_name=$(basename "$suite")
```

Comment:
This commit should be ordered before the first one as it is a mandatory prerequisite. The Git project aims for "bisectability", which means that every single commit in your patch series should always compile and test alright. With the current ordering that is not the case.